nixos/modules/services/web-apps/dolibarr/default.nix

```diff
@@ -0,0 +1,237 @@
+{ config, lib, pkgs, options, ... }:
+let
+  cfg = config.services.dolibarr;
+  app = "dolibarr";
+
+  myphp = pkgs.php;
+  version = "15.0.3";
+  configFile = "/etc/${app}/conf.php";
+  dolibarr = with pkgs; stdenvNoCC.mkDerivation {
```

Comment:
A separate package should be created for dollibar listing the proper dependencies.